sample_data/law_status.json

```diff
@@ -0,0 +1,35 @@
+[
+    {
+        "id": 1,
```

Comment:
same question as above - should we keep the id as the code is already an identifier?